templates/detail.jinja2

```diff
@@ -0,0 +1,14 @@
+{% extends "base2.jinja2" %}
+{% block body %}
+    <article class="entry">
+        <form action="{{ request.route_url('edit') }}" method="get">
+            <h2>{{ entry.title }}</h2>
+            <h2>{{ entry.date.strftime('%b. %d, %Y') }}</h2>
+            <p>{{ entry.content_md|safe }}</p>
+            <input type="hidden" name="id" value="{{ entry.id }}">
+            {% if request.authenticated_userid %}
+                <input type="submit" name="edit" value="Edit">
+            {% endif %}
```

Comment:
it's a bit odd to use a form when you aren't actually planning on submitting anything, but I can see what you're doing here.  Is there another way to pass information to the server about which entry to edit, other than using a form input to handle the job?